cmd/rm-main.go

```diff
@@ -121,9 +116,8 @@ EXAMPLES:
    8. Drop all incomplete uploads on 'jazz-songs' bucket.
       $ {{.HelpName}} --incomplete --recursive s3/jazz-songs/
 
-   9. Remove an encrypted object from s3.
-      $ {{.HelpName}} --encrypt-key "s3/ferenginar/=32byteslongsecretkeymustbegiven1" s3/ferenginar/1999/old-backup.tgz
-
+   9. Remove an encrypted object from Amazon S3 cloud storage.
+      $ {{.HelpName}} --encrypt-key "s3/sql-backups/=32byteslongsecretkeymustbegiven1" s3/sql-backups/1999/old-backup.tgz
 `,
```

Comment:
It would help to add an example with use of --encrypt flag for cp, mirror commands

Reply:
It is left out on purpose left for exploration